### DISCUSSION TITLE
Schedule room cleaning with automation

### DISCUSSION THREAD

**Discussion by DariBer** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#533] [Q&A] [answered]
First of all, I love the lovelace-xiaomi-vacuum-map-card.

I have a question, is it possible to create a schedule automation in Home Assistant, to clean a room, for example, clean kitchen every morning at 9:00.

I can't figure out what action to use and how to get that specific room

**Comment by PiotrMachowski** [comment, marked as answer]
You have to do it in a automation. To get details of service call open card editor, select a room and click "copy service call" button

**Reply by DariBer** [reply]
Ah! I totaly missed that button.
Thank you for your help
